.github/workflows/create-pr.yml

```diff
@@ -22,7 +22,7 @@ jobs:
           deno-version: 1.38.3
       - name: generate UI
         env:
-          GH_TOKEN: ${{ secrets.GH_TOKEN }}
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Nice, shall we add a comment includes this link to explain why this works?

https://docs.github.com/en/actions/security-guides/automatic-token-authentication